### PR TITLE
New package: lua-expat-1.3.0

### PR DESCRIPTION
**Pull request by raimundomartins**
Dependency of prosody, which I'll be uploading soon

### CODE REVIEW

File: srcpkgs/lua-expat/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'luaexpat'
+pkgname=luaexpat
+version=1.3.0
+revision=1
+build_style=gnu-makefile
+makedepends="lua51-devel expat-devel"
+depends="lua51 expat"
+short_desc="A SAX XML parser based on the Expat library for the Lua language"
+maintainer="Raimundo Martins <raimundoomartins@gmail.com>"
+license="GPL-compatible"
```
Comment (Gottox) [inline srcpkgs/lua-expat/template]: Please either provide a license from /usr/share/licenses or install a license file using ```vlicense```

**Comment by xtraeme** [comment]
The directory is named "lua-expat" but pkgname is named "luaexpat", please fix this.

Also the depends line is completely wrong, unless it's a pure lua module.

**Comment by asergi** [comment]
Also, makedepends on 'lua51-devel'. If so, it shoud be named 'lua51-expat'.

**Comment by xtraeme** [comment]
I agree with @asergi 

The lua packages should be renamed to match the python naming:

lua-foo -> (default lua version, 5.2)
lua5.1-foo -> for 5.1
lua5.3-foo -> for 5.3
...

**Comment by raimundomartins** [comment]
How should I proceed to fix this? Just make another commit+push and pull request?
And what about the commit message, use the same?

**Comment by xtraeme** [comment]
Reset your local repository and the force push the commit.

**Pull request closed without merging**